### PR TITLE
[AD-502] Fix bug with opening empty database.

### PR DESCRIPTION
**Pull request by Haskell-mouse**
Previously getState always tried to get the previous event
that happened before the given number of events.
This commit adds the handler for an empty database case.
Now `getState` will return special message instead of trying to get a previous event(end getting an error).

### CODE REVIEW

File: src/Data/Acid/Local.hs [hunk 1]
```diff
@@ -25,28 +26,27 @@ module Data.Acid.Local
...
 import Data.Either
-import System.FilePath                ( (</>), takeDirectory )
+import Data.IORef
+import Data.SafeCopy (SafeCopy(..), contain, primitive, safeGet, safePut)
+import Data.Serialize (runGetLazy, runPutLazy)
+import Data.Typeable (Typeable, typeOf)
+import System.Directory (createDirectoryIfMissing)
 import System.FileLock
-import System.Directory               ( createDirectoryIfMissing )
+import System.FilePath (takeDirectory, (</>))
```
Comment (artemohanjanyan) [inline src/Data/Acid/Local.hs]: Please don't change the formatting of the imports.

**Review by artemohanjanyan — COMMENTED** [review]
Please put space after `[AD-502]` in a commit message

File: src/Data/Acid/Local.hs (hunk 1 above)
Comment (Haskell-mouse) [reply src/Data/Acid/Local.hs]: Ok, reverted it.

**Review by artemohanjanyan — APPROVED** [review]

**Review by epicallan — APPROVED** [review]
LGTM

**Pull request merged**